mdaviz/utils.py

```diff
@@ -119,6 +119,35 @@ def get_md(mda_file_metadata):
     return new_metadata
 
 
+def mda2ftm(selection):
+    """Goes from  {'Y': [2, 3], 'X': 1} (mda) to {1: 'X', 2: 'Y', 4: 'Y'} (field TM)"""
```

Comment:
I don't understand this comment.  Transforms `selection` into ...(please explain)...

Reply:
added docstring